test/verifiedTests/analysis/testtuneParam/testtuneParam.m

```diff
@@ -0,0 +1,45 @@
+% The COBRAToolbox: testtuneParam.m
+%
+% Purpose:
+%     - testtuneParam tests the CPLEX parameter tuning tool of
+%       IBM CPLEX
+%
+% Author:
+%     - Marouen BEN GUEBILA 04/12/2017
+
+global CBTDIR
+global ILOG_CPLEX_PATH
+
+addpath(genpath(ILOG_CPLEX_PATH));
+
+%Load the model
+model = getDistributedModel('ecoli_core_model.mat');
```

Comment:
As mentioned above, I think there should either be a wrapper for models (which converts a model to an LPproblem, or the model should be converted in this test. Personally I think the wrapper would be better. 
I know, that a lot of people used to throw models into `solveCobraLP`, but I think thats bad practice, and for models `optimizeCbModel` should be used.

Reply:
Hey Thomas, thanks for helping with this, I addressed most of your comments. Although, I am a bit confused about the model vs lpproblem point. See if the last commit convenes.

Reply:
What I mean with LPproblem vs Model:
The COBRA Model struct has a well defined structure, which has certain properties, while a `LPproblem` structure has other properties (e.g. a `model` - normally, has an `S` matrix, while the `LPproblem` has a general `A` matrix), similarily, a model might have other fields, that influence the behavior (e.g. the C matrix which is currently being introduced), Thus, there might be essential differences between handling the structs.
Therefore I would suggest to write a second function that should be used with a model struct, and indicate that this function should only be used with an LPproblem (and actually not check if it has an `S` matrix). 
This also (imo) helps in separating LP-specific code from model specific code.

Reply:
Oh, apart from that it looks good. :)

Reply:
Good, are there examples of doing so in other functions?